src/components/Calculator.js

```diff
@@ -1,29 +1,104 @@
 import React from 'react';
+import calculate from '../logic/calculator';
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [OPTIONAL ] Kindly remove this line and let make your code professional. i do remove it on my side and check there's no linter errors. after removing that, deleting the one empty line and you code will be cool